package.json

```diff
@@ -41,23 +37,9 @@
     "eslint-plugin-jsx-a11y": "^2.2.3",
     "eslint-plugin-react": "^6.7.1",
     "jest": "^18.1.0",
-    "react": "^15.4.1",
-    "react-dom": "^15.4.1",
-    "react-waypoint": "^4.1.0",
     "rimraf": "^2.5.4",
-    "webpack": "^1.13.2"
-  },
-  "dependencies": {
-    "aws-sdk": "^2.7.5",
-    "commander": "^2.9.0",
-    "cssesc": "^1.0.0",
-    "ejs": "^2.5.2",
-    "express": "^4.14.0",
-    "mkdirp": "^0.5.1",
-    "pngjs": "^3.0.0",
-    "ps-node": "^0.1.4",
-    "selenium-webdriver": "^3.0.1",
-    "worker-loader": "^0.7.1"
+    "webpack": "^1.13.2",
```

Comment:
from what i gather, typically in a lerna monorepo you put dev deps in the root package.json so they can be shared, and then just specify the deps of each package. webpack here is a dev dep